From 2dfffdaa24f142a8f36608475d95047185b085a5 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Wed, 22 Oct 2014 11:03:04 +0800
Subject: [PATCH 39/43] fsl-echi: fix kdump failed

When kdump runs, the second kernel always gets a failure when usb_phy_clk_valid
is called, but the usb host can still work well when we skip the failure.
So return value judgement is commented off here and kdump can work well now.

Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 drivers/usb/host/ehci-fsl.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/host/ehci-fsl.c b/drivers/usb/host/ehci-fsl.c
index 419d264..8678414 100644
--- a/drivers/usb/host/ehci-fsl.c
+++ b/drivers/usb/host/ehci-fsl.c
@@ -57,10 +57,10 @@ static bool usb_phy_clk_valid(struct usb_hcd *hcd)
 {
 	void __iomem *non_ehci = hcd->regs;
 	bool ret = true;
-
+#ifndef CONFIG_CRASH_DUMP
 	if (!(in_be32(non_ehci + FSL_SOC_USB_CTRL) & PHY_CLK_VALID))
 		ret = false;
-
+#endif
 	return ret;
 }
 
-- 
1.7.5.4

